spec/FCM.spec.js

```diff
@@ -134,7 +132,7 @@ describe('FCM', () => {
       const requestData = {
         expiration_time: expirationTime,
         data: {
-          alert: 'alert',
+          alert: { body: 'alert', title: 'title' }
```

Comment:
Should we keep a test with just `alert: 'alert'`?

Reply:
Theoretically it does not need to be changed everywhere, but there needs to be at least one Android test that verifies that nested `alert` dictionary will get properly parsed. The logic behind this is that canonical iOS format is nested `alert` dictionary with `body`, `title`, and `subtitle`. Parse push adapter allows these parameters to be specified in flat request payload, but then converts them to nested `alert` for iOS anyway. 

Since iOS is way more popular than Android, I assume pushes will first be developed to target iOS, and then add Android without changing logic.

I have no insight how many people actually use this, but I got bitten by it, and I believe I am not alone, see the linked bug https://github.com/parse-community/parse-server-push-adapter/issues/212 that is not from me.

So I'm fine with having at least one test with nested dir and leaving the rest with plain `alert`.

Reply:
Yes, do we currently have both test covered, flat and nested or are any changes necessary?

Reply:
OK, I have left nested alert dir in couple places in `FCM.js` spec to make sure it really parses correctly. And removed it in others...